tethys_apps/static_finders.py

```diff
@@ -39,15 +39,15 @@ def __init__(self, *args, **kwargs):
 
         super().__init__(*args, **kwargs)
 
-    def find(self, path, all=False):
+    def find(self, path, find_all=False):
         """
         Looks for files in the Tethys apps static or public directories
         """
         matches = []
         for prefix, root in self.locations:
             matched_path = self.find_location(root, path, prefix)
             if matched_path:
-                if not all:
+                if not find_all:
```

Comment:
We can't merge this change as is without breaking support for Django 4.2. We need to find a way to support both approaches. @Sonia-96 opened a PR with the same change. See: #1172

Reply:
@shawncrawley be advised: I've merged #1172. Please update this PR accordingly.